lisp/l/extnum.l

```diff
@@ -1,4 +1,6 @@
 (defmethod ratio
- (:prin1 (&optional (strm t))
-    (format strm "~d/~d" numerator denominator))
```

Comment:
@Affonso-Gui why you removed `prin1` ?, please also update doc/test

Reply:
Removed `prin1` because 
1. `printratio` is already doing the job https://github.com/euslisp/EusLisp/blob/master/lisp/c/printer.c#L219-L226
1. `extnum.l` and its `:prin1` method are currently not being loaded in euslisp.

Reply:
@k-okada Added test code. There seems to be no ratio-related documentation to be updated. We can add some if that is the case, preferentially in https://github.com/euslisp/EusLisp/pull/359 (am planning on working on that PR this week)